wrappers/ios/Sources/Wrapper/Reader/ZXIBarcodeReader.mm

```diff
@@ -2,6 +2,7 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
+#define ZX_USE_UTF8
```

Comment:
Sorry for the back and forth but since this PR was too late for 1.4.0 and will now be part of 2.0 (I hope) this define is not necessary anymore and would merely be noise.

Reply:
All good, I removed the define